.github/dependabot.yml

```
@@ -0,0 +1,25 @@
+version: 2
+updates:
+  - package-ecosystem: "nuget"
+    directory: "/src/Dmr.Api/"
+    schedule:
+      interval: "daily"
+      time: "22:00"
+
+  - package-ecosystem: "nuget"
+    directory: "/src/DMR.Api/"
```

It probably works as-is but just for consistency:

```suggestion
    directory: "/src/Dmr.Api/"
```